{
  "comments": [
    {
      "key": {
        "uuid": "dfd5e7cf_264bc7b0",
        "filename": "inc/python",
        "patchSetId": 1
      },
      "lineNbr": 290,
      "author": {
        "id": 2472
      },
      "writtenOn": "2019-01-09T17:56:45Z",
      "side": 1,
      "message": "There was a comment on the parent patch about not needing these cases any longer, since the enabled check will return true by default now, but until we have the clean-up patch to remove these cases I think it\u0027s good to include the protection steps.",
      "revId": "3d39d5415c0910ef65fc4aecc131b4f6b04f4d00",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543",
      "unresolved": false
    }
  ]
}